ReactiveCocoa/Swift/SignalProducer.swift

```diff
@@ -360,14 +360,24 @@ extension SignalProducerType {
 		return start(Observer(observerAction))
 	}
 
-	/// Creates a Signal from the producer, then adds exactly one observer to
-	/// the Signal, which will invoke the given callback when `next` events are
-	/// received.
+	@available(*, deprecated=1.0, message="This SignalProducer may emit errors which must be handled explicitly, or observed using startWithResult:")
```

Comment:
If this is going to be deprecated, shouldn't it call through to a working implementation? Otherwise, I feel like `unavailable` would be more appropriate, because that would at least generate an error at compile time instead of runtime.